BadHabbit/BadHabbit/UserStories/MainTabBar/MainTabBarAssembly.swift

```diff
@@ -19,17 +19,29 @@ final class MainTabBarAssembly {
     
     func assemble() -> UIViewController {
         let tabVC = UITabBarController()
-        
+        tabVC.tabBar.backgroundColor = .white
         let exampleVC = exampleScreenAssembly.assembleView()
+        
+        lazy var serviceAssemblySecondScreen = ServiceAssembly(
+            requestExecutor: RequestExecutor(config: RequestExecutor.Config(baseURL: "https://www.hackingwithswift.com/samples/petitions-1.json") )
```

Comment:
URL you specified here isn't base. It's full url to specific endpoint(or file in this example). So RequestExecutor couldn't be reused in other calls.
The idea of RequestExecutor to perform calls to our backend. Backend has one domain like "www.google.com" and multiple paths like "/users" and "/cars". So to reuse RequestExecutor in multiple services we need to configure it with base domain and pass to it perform function only specific path.

Reply:
Resolved